src/storage/page_buffer.c

```diff
@@ -10178,13 +10178,10 @@ pgbuf_get_check_page_validation_level (int page_validation_level)
  *       capabilities.
  */
 DISK_ISVALID
-pgbuf_is_valid_page (THREAD_ENTRY * thread_p, const VPID * vpid, bool no_error,
-		     DISK_ISVALID (*fun) (const VPID * vpid, void *args), void *args)
+pgbuf_is_valid_page (THREAD_ENTRY * thread_p, const VPID * vpid, bool no_error)
```

Comment:
Function description also need to be modified

Reply:
fixed.